core/modules/ccontrol/queryMsg.h

```diff
@@ -37,9 +37,10 @@ int queryMsgGetCount(int session);
 
 // Python call: msg, chunkId, code, timestamp = queryMsgGetMsg(session, idx)
```

Comment:
Several files, including this one need copyright header fixed. 

core/modules/ccontrol/queryMsg.h: * Copyright 2012-2014 LSST Corporation.
core/modules/czar/python/app.py:# Copyright 2008-2014 AURA/LSST.
core/modules/czar/python/proxy.py:# Copyright 2008-2014 LSST Corporation.
core/modules/qdisp/JobStatus.cc: * Copyright 2014 LSST Corporation.
core/modules/util/testMultiError.cc: * Copyright 2014 LSST Corporation.